### PR TITLE
scalariform 0.2.1

### PR DESCRIPTION
**Pull request by markusthoemmes**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/scalariform.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 class Scalariform < Formula
   desc "Scala source code formatter"
-  homepage "https://github.com/daniel-trinh/scalariform"
-  url "https://github.com/daniel-trinh/scalariform/releases/download/0.1.7/scalariform.jar"
-  sha256 "8cb179402a7960310a8c6639a20ab94277ec3052cb75ea3ddba31265d070f0d6"
+  homepage "https://github.com/scala-ide/scalariform"
```
Comment (ilovezfs) [inline Formula/scalariform.rb]: why the switch?

Comment (markusthoemmes) [inline Formula/scalariform.rb]: The project switched its home, see https://github.com/daniel-trinh/scalariform#notice-weve-moved

Comment (ilovezfs) [inline Formula/scalariform.rb]: OK, thanks!

**Comment by ilovezfs** [comment]
@markusthoemmes Thanks for your first contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**